src/api-v1.js

```diff
@@ -584,41 +584,39 @@ api.declare({
   name: 'createAmiSet',
   deferAuth: true,
   scopes: [['aws-provisioner:manage-ami-set:<amiSetId>']],
-  input: 'create-ami-set-request.json#',
-  output: 'get-ami-set-response.json#',
   title: 'Create new AMI Set',
   stability:  base.API.stability.stable,
   description: [
     'Create an AMI Set. An AMI Set is a collection of AMIs with a single name.',
   ].join('\n'),
 }, async function (req, res) {
   let input = req.body;
-  let amiSet = req.params.amiSetId;
+  let AmiSetId = req.params.id;
```

Comment:
AmiSet is a 'class', but this isn't referring to the class.  This is referring to a string and so should be amiSetId or just id... either one is fine.